ui/app/adapters/database/connection.js

```diff
@@ -0,0 +1,76 @@
+import ApplicationAdapter from '../application';
+
+export default ApplicationAdapter.extend({
+  namespace: 'v1',
+
+  urlFor(backend, id, type = 'REST') {
```

Comment:
Should this be `REST` or `RESET`? Just see `RESET` below, wasn't sure if there were 2 or 3 values possible for `type`.

Reply:
That's a good question. I think it was intentional (anything not `ROTATE` or `RESET` falls into regular CRUD/REST bucket) but since that's the default it's better to make the default an empty string